### PR TITLE
Add Sherpa channel, pin gammapy 1.3 and fix astropy version

### PR DESCRIPTION
**Pull request by morcuended**
- Gammapy dev version cannot be installed via conda env creation 
 - Astropy version specified (0.6.1) has to be wrong. It must be 6.1
 - Sherpa is not installable via the conda forge channel, I set it to be installed via pip (within the conda env)

### CODE REVIEW

**Comment by morcuended** [comment]
One could still install Gammapy dev version with pip and the hash commit revision (`pip install git+https://github.com/gammapy/gammapy.git@6c6d15956`), but I cannot find it. I propose to set it to Gammapy 1.3 once the new version is out (it should be soon).

**Comment by facero** [comment]
sherpa can be installed via conda if adding the relevant specific channel.
Maybe adding the extra channel in third position would solve this ?
```
conda install -c https://cxc.cfa.harvard.edu/conda/sherpa -c conda-forge sherpa
```
Also I agree that we should set to 1.3 when released. It should be done these weeks.

**Comment by morcuended** [comment]
> sherpa can be installed via conda if adding the relevant specific channel. Maybe adding the extra channel in third position would solve this ?
> 
> ```
> conda install -c https://cxc.cfa.harvard.edu/conda/sherpa -c conda-forge sherpa
> ```
> 
> Also I agree that we should set to 1.3 when released. It should be done these weeks.

Ah, yes, probably with that other channel it works with conda as well

**Comment by morcuended** [comment]
@mireianievas @facero please crosscheck that astropy v6.1 is correct.

**Comment by mireianievas** [comment]
I was actually using astropy 5.3.4 I believe, probably I made a mistake there.

**Comment by mireianievas** [comment]
> sherpa can be installed via conda if adding the relevant specific channel. Maybe adding the extra channel in third position would solve this ?
> 
> ```
> conda install -c https://cxc.cfa.harvard.edu/conda/sherpa -c conda-forge sherpa
> ```
> 
> Also I agree that we should set to 1.3 when released. It should be done these weeks.

Does that sherpa ship xspec models? because the whole sherpa business was in the end just to access those specific xspec models for absorption: redden and tbabs. And since I exported them to 2D tables (with the reddening and the hydrogen column as parameters) we no longer need to access those, for any source.

**Comment by mireianievas** [comment]
Do we just specify gammapy 1.3? at some point it should be released ...

**Comment by morcuended** [comment]
> I was actually using astropy 5.3.4 I believe, probably I made a mistake there.

That's what is written in the manuscript indeed.

> Do we just specify gammapy 1.3? at some point it should be released ...

Yes, but probably better to wait until its release, and leave it commented in the meantime (or point to the gammapy main branch version to be able to install it using the env.yml file)

File: environment.yml [hunk 1]
```diff
@@ -1,10 +1,12 @@
+name: gammapy_mwl
 channels:
 - defaults
 - conda-forge
+- https://cxc.cfa.harvard.edu/conda/sherpa
 dependencies:
 - astropy=5.3.4
 - astroquery=0.4.7
-- gammapy=1.3.dev726+g6c6d15956
+# - gammapy=1.3.dev726+g6c6d15956
```
Comment (morcuended) [inline environment.yml]:
```suggestion
- gammapy=1.3
```
About to be released. Still not in conda forge, but should be there soon

**Comment by morcuended** [comment]
Even if still not in conda forge, this is ready to be merged

**Pull request merged**